docs/pages/roadmap.js

```diff
@@ -0,0 +1,694 @@
+    'status': 'unstarted',
+    'platform': 'Web',
+    'description': 'Visual updates to the Button component on the Web platform.',
+    'link': '',
+  },
+  {
+    'task': 'Button component',
+    'deadline': '',
+    'status': 'unstarted',
+    'platform': 'Android/iOS',
+    'description': 'Development of the Button component for Android and iOS.',
+    'link': '',
+  },
+  {
+    'task': 'IconButton component',
+    'deadline': '',
+    'status': 'unstarted',
+    'platform': 'Android/iOS',
+    'description': 'Development of the IconButton component for Android and iOS.',
+    'link': '',
+  },
+  {
+    'task': 'Icon component',
+    'deadline': '',
+    'status': 'unstarted',
+    'platform': 'Android',
+    'description': 'Development of the Icon component for Android.',
+    'link': '',
+  },
+  {
+    'task': 'Avatar component',
+    'deadline': '',
+    'status': 'unstarted',
+    'platform': 'Android',
+    'description': 'Development of the Avatar component for Android.',
+    'link': '',
+  },
+  {
+    'task': 'RadioGroup component',
+    'deadline': '',
+    'status': 'unstarted',
+    'platform': 'Web',
+    'description':
+      'Create new component to create accessible Radio button groups in forms, deprecate singular RadioButton.',
+    'link': '',
+  },
+  {
+    'task': 'Masonry component enhancements',
+    'deadline': '',
+    'status': 'unstarted',
+    'platform': 'Web',
+    'description': 'Develop general technical improvements to Masonry.',
+    'link': '',
+  },
+  {
+    'task': 'Device type hooks',
+    'deadline': '',
+    'status': 'inProgress',
+    'platform': 'Web',
+    'description': 'Develop functionality to unlock device-specific UI in Gestalt.',
+    'link': '',
+  },
+  {
+    'task': 'Context logging hooks',
+    'deadline': '',
+    'status': 'inProgress',
+    'platform': 'Web',
+    'description':
+      'Develop functionality to allow for generic logic to be added on component interaction.',
+    'link': '',
+  },
+  {
+    'task': 'Border color and radius design tokens',
+    'deadline': '',
+    'status': 'unstarted',
+    'platform': 'Web',
+    'description': 'Addition of tokens to support border radius and border color styling.',
+    'link': '',
+  },
+  {
+    'task': 'Color opacity design tokens',
+    'deadline': '',
+    'status': 'unstarted',
+    'platform': 'Web',
+    'description': 'Addition of tokens to support opacity styling.',
+    'link': '',
+  },
+  {
+    'task': 'Elevation design tokens',
+    'deadline': '',
+    'status': 'unstarted',
+    'platform': 'Web',
+    'description':
+      'Addition of tokens to support the styling of elevated elements within a layout.',
+    'link': '',
+  },
+  {
+    'task': 'Gestalt docs IA update',
+    'deadline': '',
+    'status': 'inProgress',
+    'platform': '',
+    'description':
+      'Update of the Gestalt docs site&rsquo;s information architecture to support pattern guidelines and mobile component documentation.',
```

Comment:
this isn't being parsed correctly:
![Screen Shot 2022-03-15 at 3 54 55 PM](https://user-images.githubusercontent.com/12059539/158485614-4fa964bd-2a97-45c5-9cc2-9dd26d3cba4e.png)

These are just strings, so you shouldn't need to use ASCII characters.

Reply:
Fixed.

Reply:
How does one add a whitespace to the end of a string that VSCode won't remove? That was the purpose of the &nbsp; entity.

Reply:
Hm…not sure if I saw the entity you're talking about, as I was just seeing what should have been `site's`. In general if you need formatting to be respected in a string in JS, you can use template literals (backticks) instead, which might be what you're looking for?

Reply:
Nevermind. I adjusted the design for the time-being to help address this.

Reply:
👍 Holler the next time you come across something like that and I'll see if I can help figure it out